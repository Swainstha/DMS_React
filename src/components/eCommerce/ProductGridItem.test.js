import testSnapFunction from '../../util/testSnapFunction';
import ProductGridItem from './ProductGridItem'
import React from 'react';
const dummy_data={
    eventList:[{id:1, name:"event1"},{id:2, name:"event2"},{id:3, name:"event3"}],
    product:{
        thumb:"thumbnail",
        name:"product1",
        price: 75,
        mrp: 80,
        offer:78,
        variant:"black",
        rating:5,
        description:"very good"
    }
}
testSnapFunction("<ProductGridItem/>", "Snapshot test for ProductGridItem",<ProductGridItem {...dummy_data}/> )
