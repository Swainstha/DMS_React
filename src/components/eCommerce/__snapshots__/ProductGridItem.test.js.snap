// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductGridItem/> Snapshot test for ProductGridItem 1`] = `ShallowWrapper {}`;
